Source/WebCore/accessibility/AXObjectCache.h

```diff
@@ -65,7 +65,7 @@ class VisiblePosition;
 class Widget;
 
 struct CharacterOffset {
-    Node* node;
+    WeakPtr<Node, WeakPtrImplWithEventTargetData> node;
```

Comment:
I would have thought `RefPtr` here instead of `WeakPtr`.

Reply:
I will check if those objects are stored long-term or not. I always worry about introducing cycles when doing that.

Reply:
I made this change.